src/ref_geo/models.py

```diff
@@ -72,6 +74,21 @@ class CorLinearGroup(db.Model):
     )
 
 
+class CorLinearArea(db.Model):
```

Comment:
Pour les many to many sans colonne additionnel, le patern est plutôt d'utiliser une `Table` plutôt qu'une classe. Il y a une raison particulière pour ce choix ? https://docs.sqlalchemy.org/en/13/orm/basic_relationships.html#many-to-many

Reply:
Oui, du coup ça oblige a faire `__table__` sur la relationship.
Je pense qu'il n'est pas nécessaire t'utiliser une classe si celle-ci n'est jamais appelée dans les routes